source/common/protobuf/utility.cc

```diff
@@ -31,7 +31,11 @@ void MessageUtil::loadFromFile(const std::string& path, Protobuf::Message& messa
 }
 
 Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& message) {
-  Protobuf::util::JsonOptions json_options;
+  Protobuf::util::JsonPrintOptions json_options;
+  // By default, proto field names are converted to camelCase when the message is converted to JSON.
+  // Setting this option makes debugging easier because it keeps field names consistent in JSON
+  // printouts.
+  json_options.preserve_proto_field_names = true;
```

Comment:
Is there any reason to not just do this inside `Protobuf::util::MessageToJsonString` ? I think we would want this for other uses also (like the admin stuff that @junr03 is working on now).